definitions/output/all/reprocess_requests.js

```diff
@@ -97,12 +107,15 @@ SELECT
   PRUNE_HEADERS(
     JSON_REMOVE(
       SAFE.PARSE_JSON(requests.summary, wide_number_mode => 'round'),
+      '$.crawlid',
       '$.firstHtml',
       '$.firstReq',
+      '$.pageid',
       '$.reqOtherHeaders',
       '$.requestid',
```

Comment:
@pmeenan I'm removing `pageid` and `requestid` from `requests.summary`?
Do you use them?

Reply:
Nah, I don't use any of the fields (or the summary tables for that matter). Those are just the unique ID's for referencing back to a specific test run.

Reply:
Would it be helpful to keep them in `meta` column? Or do you never use em (in which case we'll ditch em).

Reply:
I think @themanwithoutaplan might have depended on these IDs

Reply:
Seems like we can move forward without these variables, in the absence of a clear need.